tests/components/deconz/test_climate.py

```diff
@@ -66,27 +57,15 @@ async def test_climate_devices(hass):
     data = deepcopy(DECONZ_WEB_REQUEST)
     data["sensors"] = deepcopy(SENSORS)
     gateway = await setup_deconz_integration(hass, get_state_response=data)
-    assert "climate.thermostat" in gateway.deconz_ids
-    assert "sensor.thermostat" not in gateway.deconz_ids
-    assert "sensor.thermostat_battery_level" in gateway.deconz_ids
-    assert "climate.presence_sensor" not in gateway.deconz_ids
-    assert "climate.clip_thermostat" not in gateway.deconz_ids
-    assert len(hass.states.async_all()) == 3
-
-    thermostat = hass.states.get("climate.thermostat")
-    assert thermostat.state == "auto"
-
-    thermostat = hass.states.get("sensor.thermostat")
-    assert thermostat is None
 
-    thermostat_battery_level = hass.states.get("sensor.thermostat_battery_level")
-    assert thermostat_battery_level.state == "100"
+    assert len(hass.states.async_all()) == 2
+    assert hass.states.get("climate.thermostat").state == "auto"
+    assert hass.states.get("sensor.thermostat") is None
+    assert hass.states.get("sensor.thermostat_battery_level").state == "100"
+    assert hass.states.get("climate.presence_sensor") is None
```

Comment:
Codacy found an issue: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=6271139)